tsconfig.eslint.json

```diff
@@ -0,0 +1,13 @@
+{
+    "extends": "./tsconfig.json",
+    "include": [
+      "**/*.ts",
+      "**/*.tsx",
+      "**/*.js",
+      "capacitor.config.ts",
+      "panda.config.ts",
+      "scripts/**/*.ts",
+      "vite.config.ts",
+      "vitest.workspace.ts"
```

Comment:
I think you can remove the individual files since `**/*.ts` covers them all.

Reply:
yes, removed.